common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +96,802 @@ innovation_taming_the_skies = {
+	icon = @ursa
+
+	potential = {
+		this = culture:furbolg
+	}
+
+	unlock_maa = ursa
+
+	flag = global_maa
+}
+
+innovation_warbrave = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @warbrave
+
+	potential = {
+		this.culture_group = culture_group:tauren_group
+	}
+
+	unlock_maa = warbrave
+
+	flag = global_maa
+}
+
+innovation_war_dinosaur = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dinos
+
+	potential = {
+		this = culture:zandalari
+	}
+
+	unlock_maa = war_dinosaur
+
+	flag = global_maa
+}
+
+innovation_wolfrider = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @wolfrider
+
+	potential = {
+		this.culture_group = culture_group:orc_group
+	}
+
+	unlock_maa = wolfrider
+
+	flag = global_maa
+}
+
+
+####Early Medival
+
+innovation_baradian_marine = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @marines
+
+	potential = {
+		OR = {
+			this = culture:tirassian
+			this = culture:baradin
+		}
+	}
+
+	unlock_maa = baradian_marine
+
+	flag = global_maa
+}
+
+
+innovation_blackrock_legionnaire = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @legionnaire
+
+	potential = {
+		this = culture:blackrock
+	}
+
+	unlock_maa = blackrock_legionnaire
+
+	flag = global_maa
+}
+
+innovation_chaosborn = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @chaosborn
+
+	potential = {
+		OR = {
+			this = culture:eredar
+			this = culture:eredruin
+		}
+	}
+
+	unlock_maa = chaosborn
+
+	flag = global_maa
+}
+
+innovation_farstrider_tactics = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @farstrider
+
+	potential = {
+		OR = {
+			this = culture:high_elf
+			this = culture:blood_elf
+		}
+	}
+
+	unlock_maa = farstriders
+	unlock_maa = dragonhawk_rider
+
+	flag = global_maa
+}
+
+innovation_fel_guard = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @fel_guard
+
+	potential = {
+		OR = {
+			this = culture:annihilan
+			this = culture:nathrezim
+			this = culture:shivarra
+			this = culture:observer
+			this = culture:sayaadi
+			this = culture:satyr
+		}
+	}
+
+	unlock_maa = fel_guard
+
+	flag = global_maa
+}
+
+innovation_forsaken_ranged = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @dark_ranger
+
+	potential = {
+		this = culture:forsaken
+	}
+
+	unlock_maa = dark_ranger
+
+	flag = global_maa
+}
+
+innovation_golem_arcane = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @arcane_golem
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:draenei_group
+			this = culture:blood_elf
+			this = culture:dalaranian
+			this = culture:highborne
+			this = culture:high_elf
+		}
+	}
+
+	unlock_maa = golem_arcane
+
+	flag = global_maa
+}
+
+innovation_golem_mech = {
```

Comment:
Thinking about it, I guess it's fairer to make it cultural tech of the Dark Iron.
https://wowpedia.fandom.com/wiki/Stone_golem

Reply:
Wym here, in its current state it already is a culture tech of the Dark Iron dwarves. Do you mean to make it exclusive to them?

Reply:
> Wym here, in its current state it already is a culture tech of the Dark Iron dwarves. Do you mean to make it exclusive to them?

Yes.

Reply:
![image](https://user-images.githubusercontent.com/74790101/136703554-9dc371eb-985e-46c6-9912-77998216c66a.png)
Will give it iron dwarves as well